make.cmd

```diff
@@ -0,0 +1,23 @@
+@ECHO OFF
+where /q dotnet
+IF ERRORLEVEL 1 (
+    ECHO "ERROR: 'dotnet' not found. Please ensure you have installed .NET (the version specified in global.json)" && EXIT /b 1
+) ELSE (
+    IF "%~1" == "" (
+        dotnet fsi build.fsx --target Build
+    ) ELSE (
+        IF "%~1" == "check" (
+            dotnet fsi build.fsx --target Test
+        ) ELSE (
+            IF "%~1" == "selfcheck" (
+                dotnet fsi build.fsx --target SelfCheck
+            ) ELSE (
+                IF "%~1" == "docs" (
+                    dotnet fsi build.fsx --target Docs
+                ) ELSE (
+                    ECHO "Target was not recognized" && EXIT /b 1
+                )
+            )
+        )
+    )
+)
```

Comment:
Shouldn't we just pass the argument as is `-t %~1` without this long match?

Reply:
I thought about this, but there's one target that doesn't match: check maps to Test. And I decided to make it `make check` because it is a convention to call the unit tests target this way in the Makefile world.